packages/tasks/src/local-apps.ts

```diff
@@ -100,12 +100,14 @@ export const LOCAL_APPS = {
 		docsUrl: "https://drawthings.ai",
 		mainTask: "text-to-image",
 		macOSOnly: true,
-		comingSoon: true,
-		/**
-		 * random function, will need to refine the actual conditions:
-		 */
-		displayOnModelPage: (model) => model.library_name === "diffusers" && model.pipeline_tag === "text-to-image",
-		deeplink: (model) => new URL(`drawthings://open_from_hf?model=${model.id}`),
+		displayOnModelPage: (model) => model.library_name === "diffusers" && (model.pipeline_tag === "text-to-image" || model.tags.includes("lora")),
+		deeplink: (model) => {
+			if (model.tags.includes("lora")) {
+				return new URL(`https://drawthings.ai/import/diffusers/pipeline.load_lora_weights?repo_id=${model.id}`);
```

Comment:
It seems this only works in Safari (Universal Link). I will include scheme based link support in the next update and also publish our fallback page soon (design ready, need to code up the HTML page).

The scheme based link would look like: `drawthings:///import/diffusers/pipeline.from_pretrained?repo_id=${model.id}` (similarly for LoRA weights). (note that it is /// because the host name is empty).

Our fallback page available at https://drawthings.ai/import/diffusers/pipeline.load_lora_weights will look roughly like this:
<img width="993" alt="Screenshot 2024-05-15 at 4 14 32 PM" src="https://github.com/huggingface/huggingface.js/assets/127987/0b3edc9b-e950-4cba-a938-1abe7fa48755">

Reply:
very cool!

should we merge that PR i.e. we send users to your website and from there, you attempt to open the drawthings:/// url yourselves, correct?

Or do you want us to open the custom protocol ourselves (and fallback to the https:// page)?

Reply:
will merge this one so we can already test in production, but feel free to open a new PR or suggest changes